go/client/fork_server.go

```diff
@@ -140,6 +140,12 @@ func makeServerCommandLine(cl libkb.CommandLine, forkType keybase1.ForkType) (ar
 		}
 	}
 
+	// If there is no explicit log file add one when autoforking.
+	// otherwise it was added in the previous block already.
+	if cl.GetLogFile() == "" {
```

Comment:
Can you plumb `G` all the way through and then call `g.Env.GetLogFile()`? That way we can also pick up the log file from the environment too.